### PR TITLE
About Yoroi block

### PR DESCRIPTION
**Pull request by ASUKA1006**
1) Removed the scrollbar for the footer (size was too small to fit SVG)
2) Adds About Us to the settings page

- [ ] Need SVG logo for EMURGO
- [ ] Yoroi logo is plain white so it is hard to see
- [ ] Review these are the links we want on the About Us
- [ ] Maybe left-align instead of center?
- [ ] #289 Added a media query with a size specifically for the add wallet page (5-element rows and full width of app). This means the text for our icons disappears even when there is a lot of space left. Probably we need to replace this media query with something smarter.

screen shot 2019-02-10 at 3 13 12 am

### CODE REVIEW

**Comment by sarveshwar-singh** [comment]
(Was busy with something else, just followed: https://github.com/Emurgo/yoroi-frontend/issues/263 https://github.com/Emurgo/yoroi-frontend/pull/269 )
1. I strongly believe we should make it as new item rather than fitting somewhere else. 
Remove from `General`, add new item as `About Yoroi`, move content to `About Yoroi`
image

2. Add Version, Commit, Network (can find logic here: https://github.com/Emurgo/yoroi-frontend/blob/06e9e353da26609435af36e0b085e80f32a4f9b6/app/utils/logging.js#L50), with standard Yoroi styling (fonts)
image

3. Change `Yoroi version` to `Yoroi changelog` (@SebastienGllmt  or any better)

4. 
>  Need SVG logo for EMURGO

try out: https://github.com/Emurgo/yoroi-mobile/blob/a3d72218b1e63f6362152aae2f03c8763c168795/src/assets/EmurgoIcon.js#L10

5. 
> * Yoroi logo is plain white so it is hard to see

Yes, we can change it to black.
Following is how to customize it:
```
.yoroiLogo {
  & > svg {
    display: block;
    width: 100px;
    margin: 0 114px;
    & > path {
      fill: var(--theme-button-primary-background-color);
    }
  }
}
```  
https://github.com/Emurgo/yoroi-frontend/blob/develop/app/components/loading/Loading.scss
There is only white color logo is there in repo but we customize it depends upon need.

6. 
> * #289 Added a media query with a size specifically for the add wallet page (5-element rows and full width of app). This means the text for our icons disappears even when there is a lot of space left. Probably we need to replace this media query with something smarter.

was not aware the you are going to reuse that here.

I would suggest:
A. 
move this: https://github.com/Emurgo/yoroi-frontend/tree/develop/app/components/footer
to: https://github.com/Emurgo/yoroi-frontend/tree/develop/app/components/widgets
and give a more generic name rather than `FooterItem`
(`FooterItem` is not appropriate for Setting page)

B.
move this: https://github.com/Emurgo/yoroi-frontend/blob/develop/app/containers/footer/AddWalletFooter.js
to: https://github.com/Emurgo/yoroi-frontend/tree/develop/app/components/footer

also move `media query` from `FooterItem`(current) to here.


can directly use: `AddWalletFooter` at https://github.com/Emurgo/yoroi-frontend/blob/fe956dc9b035232ed653a1619a2839ff64010ade/app/containers/wallet/WalletAddPage.js#L107

7.
`app/containers/footer/GeneralSettingsFooter.js` should no be in footer from start.
It should should have been inside `app/containers/settings` but if you do `#1` and `#6` as i mentioned there will be new file `app/containers/settings/AboutYoroiPage.js` and mostly we don't need `app/containers/footer/GeneralSettingsFooter.js`

**Comment by ASUKA1006** [comment]
For the request 1, 
I removed  `About Yoroi` from `General` and added a new item for `About Yoroi`. Also, I moved that content from `General` to it.
Now, the pages look like this.

[About Yoroi page]
screen shot 2019-02-13 at 12 23 59 am

[General page]
screen shot 2019-02-13 at 12 22 55 am

**Comment by SebastienGllmt** [comment]
Here is the list from the marketing team for the URLs to include:

1) Yoroi Facebook needs to be added.
2) Yoroi Twitter and Facebook should go first in order
3) Delete Yoroi Version (replace with suggestion from Shin)
4) Replace "EMURGO Blog" with just "EMURGO" to emurgo.io
5) Add Medium blog link
5) Delete the telegram link in the "About Us" and put in the "Support" page instead. Rename as "Telegram Tech Support."

File: app/components/layout/GridFlexContainer.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.component {
```
Comment (SebastienGllmt) [inline app/components/layout/GridFlexContainer.scss]: This is all unused

File: app/components/layout/HorizontalFlexContainer.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .component {
   width: 100%;
+  height: 100%;
```
Comment (SebastienGllmt) [inline app/components/layout/HorizontalFlexContainer.scss]: I don't think this is necessary?

**Comment by SebastienGllmt** [comment]
Hey @ASUKA1006 thanks for the fixes. Your latest commits didn't change the color of the Yoroi icon (but it was close!) so I fixed that for you. Here is what it looks like now
image

I think this is good enough for a v1 and we can fix the other issues later after we release `1.3.0`. Thoughts?

**Comment by SebastienGllmt** [comment]
image
Just pushed some small fixes for @vantuz-subhuman

**Pull request merged**